pydantic/utils.py

```diff
@@ -268,3 +269,16 @@ def resolve_annotations(raw_annotations: Dict[str, AnyType], module_name: Option
 
 def is_callable_type(type_: AnyType) -> bool:
     return type_ is Callable or getattr(type_, '__origin__', None) is Callable
+
+
+def _check_classvar(v: AnyType) -> bool:
+    return type(v) == type(ClassVar) and (sys.version_info < (3, 7) or getattr(v, '_name', None) == 'ClassVar')
+
+
+def is_classvar(ann_type: AnyType) -> bool:
```

Comment:
again this seems to be messed up due to a rebasing problem.

Reply:
Should be ok now